using_images/db_images/mysql.adoc

```diff
@@ -26,42 +29,53 @@ This image comes in two flavors, depending on your needs:
 
 The RHEL 7 image is available through Red Hat's subscription registry via:
 
-****
-`$ docker pull registry.access.redhat.com/openshift/mysql-55-rhel7`
-****
+----
+$ docker pull registry.access.redhat.com/openshift/mysql-55-rhel7
+----
 
 *CentOS 7 Based Image*
 
 This image is available on DockerHub. To download it:
 
-****
-`$ docker pull openshift/mysql-55-centos7`
-****
+----
+$ docker pull openshift/mysql-55-centos7
+----
 
-To use these images, you can either access them directly from those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
+To use these images, you can either access them directly from these
+registries or push them into your OpenShift docker registry. Additionally,
+you can create an ImageStream that points to the image,
+either in your docker registry or at the external location. Your OpenShift
+resources can then reference the ImageStream. You can find
+https://github.com/openshift/origin/tree/master/examples/image-streams[example]
+ImageStream definitions for all the provided OpenShift images.
 
 == Configuration and Usage
 
 === Initializing the Database
 
-The first time you use the shared volume, the database is created along with the database administrator user and the MySQL root user (if you specify the `*MYSQL_ROOT_PASSWORD*` environment variable).  Afterwards, the MySQL daemon starts up. If you are re-attaching the volume to another container then the database, database user and the administrator user are not created, and the MySQL daemon starts.
+The first time you use the shared volume, the database is created along with 
+the database administrator user and the MySQL root user (if you specify the 
+`*MYSQL_ROOT_PASSWORD*` environment variable).  Afterwards, the MySQL daemon 
+starts up. If you are re-attaching the volume to another container then the 
+database, database user and the administrator user are not created, and the 
```

Comment:
s/database, database user and the administrator user/database, database user, and the administrator user